src/stringformat-private.h

```diff
@@ -44,6 +44,7 @@ struct _StringFormat {
 	StringFormatFlags	formatFlags;
 	StringTrimming		trimming;
 	StringDigitSubstitute	substitute;
+	LANGID		language;
```

Comment:
please fix indendation so it lines up with the others

Reply:
I am the worst at indentation, sorry! For some reason VS code sometimes gets it right but often doesn't. It also doesn't seem to be able to line up tabs correctly

Reply:
yeah I see the same in VS Code. nevermind ❤️